framework/src/org/checkerframework/common/basetype/flow.astub

```diff
@@ -7,6 +7,8 @@ package java.lang;
 
 public final class Class<T> {
     @Pure public String getName();
+    @Pure public java.lang.reflect.Method getMethod(String a1, Class<?> ... a2);
```

Comment:
Making these two methods pure makes reflective code easier to use, as less information is lost.